app/views/admin/check_regional_records.html.erb

```diff
@@ -29,6 +29,11 @@
       huge).
     </p>
 
+    <%= alert :warning do %>
+      This script relies on auxiliary tables which are computed as part of <%= link_to "running CAD", admin_compute_auxiliary_data_path, target: :_blank %>.
+      <b>It will not deliver accurate results for specific competitions if CAD hasn't been run after the results posting.</b>
```

Comment:
Maybe replace "after the results posting" with "after the 'import results' step", so that it won't get confused with last step?